code/datums/components/twohand.dm

```diff
@@ -0,0 +1,92 @@
+/obj/item/proc/is_twohandable() // Returns TRUE if an object has a two-handing component. FALSE otherwise.
+	if(GetComponent(/datum/component/twohand))
+		return TRUE
+
+/obj/item/proc/is_twohanded() // Returns TRUE if an item is wielded with two hands. FALSE otherwise.
```

Comment:
shouldn't this be `is_wielded`?

Reply:
That would be a matter of semantics. two-handed is a more descriptive title given what we're looking for, "wielded" is slightly ambiguous.